specification/botservice/resource-manager/Microsoft.BotService/preview/2017-12-01/botservice.json

```diff
@@ -683,6 +683,56 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.BotService/locations/{location}/operationresults/{operationId}": {
+      "get": {
+        "x-ms-examples": {
+          "Get Server Details": {
+             "$ref": "./examples/operationResults.json"
+             }
+        },
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "OperationResult_Get",
+        "description": "List the result of the specified operation.",
+        "parameters": [
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The region name which the operation will lookup into. including 'global'"
+          },
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The target operation Id."
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "If the operation completed successfully, the service should return 200 (OK)."
+          },
+          "202": {

```

Comment:
On similar lines, not sure if the service is supposed to return a `202` response. 202 status code corresponds to `accepted`, i.e., the operation as been accepted but not completed. A non-long running GET operation MUST finish and return right away.
Please confirm is the service has undergone an ARM/API review

Reply:
the above pattern is directly taken from azure-rest-api-specs\specification\analysisservices\resource-manager\Microsoft.AnalysisServices\stable\2016-05-16\analysisservices.json (please see line 425)